src/components/mgt-people/mgt-people.ts

```diff
@@ -250,6 +265,17 @@ export class MgtPeople extends MgtTemplatedComponent {
    * @memberof MgtPeople
    */
   protected renderPerson(person: MicrosoftGraph.User | MicrosoftGraph.Person | MicrosoftGraph.Contact): TemplateResult {
+    // Get person presence from _peoplePresence to pass to mgt-person
+    let personPresence = null;
+    for (const presenceObj of this._peoplePresence) {
```

Comment:
You can avoid this for loop by using an object to store the presence by the id of the user

Reply:
_peoplePresence would still be an array of object though. Are you suggesting using Array.filter() or find()?